apps/app/src/components/UnlockStore.tsx

```diff
@@ -48,23 +60,32 @@ const UnlockStore: Component = () => {
             onInput={(e: any) => {
               setPassword(e.target.value);
               setError(false);
-            }}
-          />
+            } } />
           <Show
             when={error()}
             fallback={<FormHelperText>{t()("UNLOCK_TEXT")}</FormHelperText>}
           >
             <FormErrorMessage>{t()("WRONG_PASSWORD")}</FormErrorMessage>
           </Show>
         </FormControl>
-        <HStack justifyContent="flex-end">
+        <HStack  spacing="24px" justifyContent="flex-end">
```

Comment:
formatting (double space)
```suggestion
        <HStack spacing="24px" justifyContent="flex-end">
```

Reply:
also, ive been using hope-ui variables for spacings: using something like `$4` instead of `24px` would make the code more consistent